fiducial_len_override causing a CV error

Upon setting fiducial_len_override, aruco_detect throws errors even after noting that it has detected all 4 markers in the image.

```
[INFO] [1560215349.612089574]: Detected 4 markers
OpenCV Error: Assertion failed (markerLength > 0) in getSingleMarkerObjectPoints, file /tmp/binarydeb/ros-kinetic-aruco-detect-0.11.0/src/aruco_detect.cpp, line 127
[ERROR] [1560215349.612601219]: cv exception: /tmp/binarydeb/ros-kinetic-aruco-detect-0.11.0/src/aruco_detect.cpp:127: error: (-215) markerLength > 0 in function getSingleMarkerObjectPoints
```

Here is the relevant part of my launch file:

```
<node pkg="aruco_detect" type="aruco_detect" name="aruco_detect" output="screen">
        <param name="fiducial_len" value="0.1"/>
        <param name="fiducial_len_override" value="0-3: 0.1, 4: 0.15"/>
        <param name="dictionary" value="0"/>
        <param name="publish_images" value="true"/>
        <remap from="/camera" to="/cv_camera/image_raw"/>
        <remap from="/camera_info" to="/cv_camera/camera_info"/>
</node>
```

My use of fiducial_len_override looks very similar to the documentation so I am confused where the error stems from.

--- a/aruco_detect/src/aruco_detect.cpp
+++ b/aruco_detect/src/aruco_detect.cpp
@@ -530,7 +530,7 @@ FiducialsNode::FiducialsNode() : nh(ros::NodeHandle("~")), it(nh)
         std::vector<std::string> parts;
         boost::split(parts, element, boost::is_any_of(":"));
         if (parts.size() == 2) {
-            double len = std::stod(parts[0]);
+            double len = std::stod(parts[1]);
             std::vector<std::string> range;
             boost::split(range, element, boost::is_any_of("-"));
             if (range.size() == 2) {

